client/js/app/components/explorer/visualization/keen_viz_config.js

```diff
@@ -1,13 +1,28 @@
 var React = require('react');
+var classNames = require('classnames');
 
 var KeenVizConfig = React.createClass({
 
+  propTypes: {
+    onCloseClick: React.PropTypes.func.isRequired,
+    hidden: React.PropTypes.bool.isRequired
+  },
+
   render: function() {
+    var panelClasses = classNames({
+      'viz-config-panel': true,
+      'hide': this.props.hidden
+    });
+
     return (
-      <div></div>
+      <div className={panelClasses}>
+        <a href="#" className="close-btn pull-right" onClick={this.props.onCloseClick}>
+          <span className="icon glyphicon glyphicon glyphicon-remove-circle no-margin"></span>
+        </a>
+        <div className="alert alert-info">Hello, World.</div>
```

Comment:
Are these classes just here for the time being? I'm not sure if `alert` is appropriate for the config box.